mautic/mautic
# [UI] Assign links, body and control-label text color to tokens

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
# Edit

Since we didn't get to a conclusion and here [UI] [Accessibility features in user profile](https://github.com/mautic/mautic/pull/13887) I add the underlined links as a feature, I used this PR to just assign the links and body text color to the right tokens.

Contrast Ratio goes from 9.62:1 to 18.1:1 for text color. Both passed AAA WCAG guidelines, but the new darker color feels like a good improvement for the eyes

Links remained the same for now...



Before:

![image](https://github.com/mautic/mautic/assets/116097999/cb9e0d60-dcfa-409c-b6d5-2ea0645a3fc3)

![image](https://github.com/mautic/mautic/assets/116097999/59fe871b-be93-4f71-b702-3fa3276b96dd)


After:

![image](https://github.com/mautic/mautic/assets/116097999/f1c77642-ff17-484f-9569-fb03a95454fc)

![image](https://github.com/mautic/mautic/assets/116097999/323137fe-b634-497e-8856-bee6f4e587c1)







#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Click on the top navbar links
3. Click on the menu items
4. Click on links you find
5. Look at the general text color
